queries/thrift/highlights.scm

```diff
@@ -27,15 +27,27 @@
 
 ; Fields
 
-(field) @field
+(field_identifier) @field
 
 ; Parameters
 
-(parameter) @parameter
+(param_identifier) @parameter
 
 ; Variables
 
 (identifier) @variable
+(annotation_identifier) @variable
+
+; Constants
+
+(const_identifier) @constant
+(enum_identifier) @constant
+(enum_member) @constant
```

Comment:
Shouldn't `enum_identifier` be `@type` instead?

Reply:
it's possible you're confusing the ident as enum Foo { ... } where Foo is said ident, that's not the case 

I labeled enum_member differently because they can be used when assigning consts


```thrift
enum Numberz {
  NAN,
  ONE = 1,
  TWO = 2,
  ...
}

const Numberz myNumberz = Numberz.ONE;
```

NAN, ONE, TWO are the enum_identifiers

in Numberz.ONE, ONE is the "enum_member", should've been named accessor maybe? or just enum_identifier too